cogs5e/models/sheet/base.py

```diff
@@ -172,6 +185,19 @@ def default(cls, base_stats: BaseStats = None):
         for skill in SKILL_NAMES:
             skills[skill] = Skill(base_stats.get_mod(SKILL_MAP[skill]))
         return cls(skills)
+    
+    @classmethod
+    def custskills(cls, origskills, custstats: BaseStats = None, addiskls = None):
+        """Returns a skills object with skills set to default values, based on stats."""
```

Comment:
docs: this doesn't appear to be accurate (this is the docs of `Skills.default`)

Reply:
I'm not 100% exactly what this method is doing. It's worth breaking out much of the nested logic into multiple lines to improve readability.

Reply:
It appears to be constructing a new `Skills` given proficiencies and bonuses defined by another `Skills` object, recalculating each `Skill.value` with the appropriate modifier given by `custstats`?

I'm not sure what the purpose of `addiskills` is here. This looks like it should be broken into two or more methods.

Reply:
Addiskills was a planned addition of a player being able to set skill proficiencies individually, but I didn't get around to it. Also I forgot to update the doc.